src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private final int firstLeg;
+    private final int secondLeg;
+    private final int hypotenuse;
+
+    public RightTriangle(String color, int firstLeg, int secondLeg) {
+        super(color);
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+        this.hypotenuse = (int) (Math.sqrt(Math.pow(firstLeg,2))
+                               + Math.sqrt(Math.pow(secondLeg,2)));
```

Comment:
The calculation of the hypotenuse seems incorrect. The Pythagorean theorem states that the square of the hypotenuse is the sum of the squares of the other two sides. You should sum the squares first before applying the square root.